src/config-schema.ts

```diff
@@ -239,11 +226,177 @@ export const configSchema = {
             target: 'body',
             title:
               "Great job! You've completed the tutorial. Now, take a moment to explore the Patient Chart View and discover all its features. Feel free to navigate around and get comfortable with the layout. If you need to return to the home page, just click the close button in the top right corner. Happy exploring!",
-            disableOverlayClose: true,
             placement: 'center',
           },
         ],
       },
+      {
+        title: 'Patient Lists',
+        description: 'Patient list is a dynamically populated list of patients that can be used to organize patients for easier management.',
```

Comment:
Maybe something like this?
"This tutorial will guide you through creating and managing patient lists to help organize and track your patients more effectively."

Reply:
This makes sense!